src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -136,12 +136,12 @@ def get(self, request: Request, group) -> Response:
         if request.GET.get("threshold"):
             similar_issues_params.update({"threshold": float(request.GET["threshold"])})
 
-        results = get_similar_issues_embeddings(similar_issues_params)
-
         extra: dict[str, Any] = dict(similar_issues_params.copy())
         extra["group_message"] = extra.pop("message")
         logger.info("Similar issues embeddings parameters", extra=extra)
 
+        results = get_similar_issues_embeddings(similar_issues_params)
+
```

Comment:
@jangjodi  Would it make sense to add another log after for some data about the result of the API call? Maybe status codes or number of results, etc. Unless you don't care about those logs in which case no change is needed.

Reply:
thanks for the suggestion! we're showing the results in the UI and we have an analytic event after that shows some info, so I don't think we need more